src/components/Timeline/Message/MessageActions.tsx

```diff
@@ -26,148 +31,228 @@ export interface MessageActionsProps {
+                            <Link
+                                onClick={() => {
+                                    setStreamListOpen(true)
+                                }}
+                                underline="hover"
+                                sx={{
+                                    fontweight: '400',
+                                    fontSize: '12px',
+                                    color: 'text.secondary'
+                                }}
+                            >
+                                +{props.msgstreams.length - 1}
+                            </Link>
+                        )}
+                    </Box>
+                )}
+            </Box>
+            <Collapse in={streamListOpen} collapsedSize={0}>
+                <Box
                     sx={{
-                        p: '0',
-                        color: props.theme.palette.text.secondary
-                    }}
-                    onClick={(e) => {
-                        props.setMessageAnchor(e.currentTarget)
+                        display: 'flex',
+                        justifyContent: 'space-between'
                     }}
                 >
-                    <MoreHorizIcon sx={{ fontSize: { xs: '70%', sm: '80%' } }} />
-                </IconButton>
-            </Box>
-            <Box display="flex">
-                {/* right */}
-                {props.msgstreams.map((e) => (
-                    <Link
-                        key={e.id}
-                        underline="hover"
-                        sx={{
-                            fontweight: '400',
-                            fontSize: '12px',
-                            color: 'text.secondary'
-                        }}
-                        href={'/#' + e.id}
-                    >
-                        {`%${
-                            e.payload.body.name.length < 15
-                                ? (e.payload.body.name as string)
-                                : (e.payload.body.name.slice(0, 2) as string) + '...'
-                        }`}
-                    </Link>
-                ))}
-            </Box>
-        </Box>
+                    <Box sx={{ display: { xs: 'block', sm: 'none' } }}>
+                        {props.msgstreams.map((e) => (
+                            <Link
+                                key={e.id}
+                                underline="hover"
+                                sx={{
+                                    fontweight: '400',
+                                    fontSize: '12px',
+                                    color: 'text.secondary'
+                                }}
+                                href={'/#' + e.id}
+                            >
+                                {`%${e.payload.body.shortname as string}`}
+                            </Link>
+                        ))}
+                    </Box>
+                </Box>
```

Comment:
最終的に使われてなさそうなBoxのネストがあるのがちょっと気になるかも？

後試しにgapいれてみた(0.5でもいいかも)

```suggestion
                <Box
                    sx={{
                        display: { xs: 'flex', sm: 'none' },
                        gap: 1
                    }}
                >
                    {props.msgstreams.map((e) => (
                        <Link
                            key={e.id}
                            underline="hover"
                            sx={{
                                fontweight: '400',
                                fontSize: '12px',
                                color: 'text.secondary'
                            }}
                            href={'/#' + e.id}
                        >
                            {`%${e.payload.body.shortname as string}`}
                        </Link>
                    ))}
                </Box>

```